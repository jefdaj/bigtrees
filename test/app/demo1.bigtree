F 1 ZTgwZDhlZDM3NDUxN2Uy 1713640055 file1.txt
F 2 Y2ZjOTQ5NGVjMTQ4M2I2 1713640055 file3.txt
F 3 OGMwODk5YWZhOTllMWVh 1713640055 file2.txt
D 2 ZDUzYWU5MWIyNTAwNjU5 1713640055 folder2
D 1 NDEzYWJiZjYyZDY3MmI1 1713640055 folder1
D 0 OTY2NjU5NzE5MjczZGMx 1713640055 demo1
