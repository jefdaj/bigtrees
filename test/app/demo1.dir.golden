F 1 ZTgwZDhlZDM3NDUxN2Uy file1.txt
F 2 Y2ZjOTQ5NGVjMTQ4M2I2 file3.txt
F 3 OGMwODk5YWZhOTllMWVh file2.txt
D 2 ZDUzYWU5MWIyNTAwNjU5 folder2
D 1 NDEzYWJiZjYyZDY3MmI1 folder1
D 0 OTY2NjU5NzE5MjczZGMx demo1.dir
